.github/workflows/ci-build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# AppVeyor and Drone Continuous Integration for MSYS2
+# Author: Renato Silva <br.renatosilva@gmail.com>
+# Author: Qian Hong <fracting@gmail.com>
+
+# Configure
+cd "$(dirname "$0")"
+source 'ci-library.sh'
+mkdir artifacts
+git_config user.email 'ci@msys2.org'
+git_config user.name  'MSYS2 Continuous Integration'
+git remote add upstream 'https://github.com/MSYS2/MINGW-packages'
+git fetch --quiet upstream
+# So that makepkg auto-fetches keys from validpgpkeys
+mkdir -p ~/.gnupg && echo -e "keyserver keyserver.ubuntu.com\nkeyserver-options auto-key-retrieve" > ~/.gnupg/gpg.conf
+# reduce time required to install packages by disabling pacman's disk space checking
+sed -i 's/^CheckSpace/#CheckSpace/g' /etc/pacman.conf
+
+# Detect
+list_commits  || failure 'Could not detect added commits'
+list_packages || failure 'Could not detect changed files'
+message 'Processing changes' "${commits[@]}"
+test -z "${packages}" && success 'No changes in package recipes'
+define_build_order || failure 'Could not determine build order'
+
+# Build
+message 'Building packages' "${packages[@]}"
+execute 'Approving recipe quality' check_recipe_quality
```

Comment:
I am not sure what all this script is doing (referring to `define_build_order` and `check_recipe_quality`) even with `ci-library.sh` included, and I do not believe that we maintainers want to left to maintain it.

Reply:
I am happy to drop this bit, unforunately, it can't be tested until it has been merged, which is a bit of a catch-22 situation anyway.

This is pacman/makepkg stuff. I used the canonical MSYS2 ci setup.